mne_bids/read.py

```diff
@@ -231,6 +231,18 @@ def _handle_channels_reading(channels_fname, bids_fname, raw):
         # Try to map from BIDS nomenclature to MNE, leave channel type
         # untouched if we are uncertain
         updated_ch_type = bids_to_mne_ch_types.get(ch_type, None)
+
+        if updated_ch_type is None:
+            # XXX Try again with uppercase spelling – this should be removed
```

Comment:
I am going to ask the dreaded question: where is the unit test?

Reply:
okokok, will add one :)

Reply:
Done in 3818404